src/app/shared/components/template/template-component.ts

```diff
@@ -154,8 +150,7 @@ export class TemplateComponent implements OnInit, AfterContentInit, ITemplateRow
     row: FlowTypes.TemplateRow
   ) {
     const viewContainerRef = this.tmplComponentHost.viewContainerRef;
-    const factory = this.componentFactoryResolver.resolveComponentFactory(component);
-    const componentRef = viewContainerRef.createComponent(factory);
+    const componentRef = viewContainerRef.createComponent(component);
```

Comment:
minor refactor required for how angular handles dynamic components. Tested working (by loading any template page which will in turn load dynamic components)